<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuanLiYuan</class>
 <widget class="QWidget" name="GuanLiYuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园寻路系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="School.qrc">
    <normaloff>:/School/Picture/tb.png</normaloff>:/School/Picture/tb.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>添加删除系统（管理员）</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>770</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>当前用户：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="CurrentYH">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="Jianjie">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>150</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="xg_Button">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>470</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改简介</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="School.qrc"/>
 </resources>
 <connections/>
</ui>
